core/src/main/java/org/apache/accumulo/core/volume/Volume.java

```diff
@@ -38,26 +38,25 @@
   String getBasePath();
 
   /**
-   * Convert the given Path into a Path that is relative to the base path for this Volume
+   * Convert the given child path into a Path that is relative to the base path for this Volume. The
+   * supplied path should not include any scheme (such as <code>file:</code> or <code>hdfs:</code>),
+   * and should not contain any relative path "breakout" patterns, such as <code>../</code>. If the
+   * path begins with a single slash, it will be preserved while prefixing this volume. If it does
+   * not begin with a single slash, one will be inserted.
```

Comment:
It looks like you are just clarifying what should be in the string parameter and did not change the behavior correct?

Reply:
The only behavior change is the stricter checking of params. Previously, it wouldn't work right... if you passed in a param that had a leading slash, it would not prefix it correctly.